.github/workflows/devin-slack-trigger.yml

```
@@ -51,7 +51,8 @@ jobs:
           script: |
             const issue = context.payload.issue;
 
-            const message = `[${issue.title}]\n<@${process.env.DEVIN_USER_ID}> Please handle this issue. First, create a plan and share it before implementing. Think in English, respond in Japanese.\n${issue.html_url}`;
+            const message = `!swe
+[${issue.title}]\n<@${process.env.DEVIN_USER_ID}> Please handle this issue. First, create a plan and share it before implementing. Think in English, respond in Japanese.\n${issue.html_url}`;
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)